roles/veyepar-encoder/tasks/main.yml

```diff
@@ -3,36 +3,65 @@
   apt:
     name: "{{ item }}"
   with_items:
+  - python-psycopg2
```

Comment:
maybe.  I'll assume you mean the local one that is setup elseware here with requirements.txt, not one on another box.

 I waffle between using apt vs pip to install things,  and --site-packages,  and often I use both.